Feature/82

byyyyyyyyy opened the pull request:
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?
Weapons:
![image](https://user-images.githubusercontent.com/88365848/143290939-b7de6315-5f99-4d88-bea4-9ceddb0befcf.png)

Shields:
![image](https://user-images.githubusercontent.com/88365848/143290992-ebdb27d3-70f3-404f-b292-0f0b40e9d67f.png)
![image](https://user-images.githubusercontent.com/88365848/143291013-41ba9380-b2cb-4203-bc34-a487d271b47d.png)


Characters:
![image](https://user-images.githubusercontent.com/88365848/143291090-f49037d0-8416-45f6-ac24-e03cad29664a.png)
![image](https://user-images.githubusercontent.com/88365848/143291129-326e9486-f88c-4037-9387-72af8a4a4311.png)


### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

added sorting and filtering for abi weapons, characters, shields, skill shop. 

(insert description here)

wintersummermint reviewed (commented):
just remove unused commented code

byyyyyyyyy commented:
> just remove unused commented code

hi sir. i've removed the commented code but only the ones that i've added, i might remove some important commented codes if i remove them. maybe some of them still has a use for future developments.
